ext/native/thread/executor.cpp

```diff
@@ -10,7 +10,10 @@ void SameThreadExecutor::Run(std::function<void()> func) {
 }
 
 void NewThreadExecutor::Run(std::function<void()> func) {
-	std::thread(func).detach();
+	std::thread thr = std::thread(func);
+#ifndef HAVE_LIBNX
+    thr.detach();
```

Comment:
Ah, I missed this one.  Seems like we could just make it a simple property of `NewThreadExecutor`?

-[Unknown]

Reply:
Either way works for me ;)